packages/admin/src/index.ts

```diff
@@ -1,31 +1,123 @@
-// Export Folders
-export * from "./helpers";
-export * from "./router";
-export * from "./stack";
-export * from "./table";
-// Export ...
-export * from "./fetchProvider";
-export * from "./fileIcons";
-export * from "./form";
-export * from "./mui";
-export * from "./common";
-
-// Export Files
-export * from "./buildRestMutation";
-export * from "./DeleteMutation";
-export * from "./DirtyHandler";
-export * from "./DirtyHandlerApiContext";
-export * from "./EditDialog";
-export * from "./EditDialogApiContext";
-export * from "./FinalForm";
-export * from "./FormMutation";
-export * from "./RouterTabs";
-export * from "./Selected";
-export * from "./Selection";
-export * from "./SelectionApi";
-export * from "./SelectionRoute";
-export * from "./Tabs";
-
-export * from "./error";
-
-export * from "./hooks";
+export { IWindowSize, useWindowSize } from "./helpers/useWindowSize";
```

Comment:
i like that most default exports are gone 👍 

I am still not 100% sure if it's a good idea to export everything from the root index.ts file, it's kind of, throw everything in here. Sounds like we will get a lot of conflicts here, and in my opionion it's confusing and unclear.

But that was our decision - it's ok for me

Reply:
Yes that's a good point.

Reply:
The main advantage I see is that we have a single file where we see all exports. Easy to review and make sure the API stays clean.

We will see if too many conflicts appear (I'm also a bit afrait of that) - if that is a real problem we discuss it again....